### PR TITLE
RMET-446 FileViewer Plugin - Create JS layer

### PR DESCRIPTION
**Pull request by alexgerardojacinto**
## Description

Added JS methods that match the functionalities identified in the task.

## Context

Closes: https://outsystemsrd.atlassian.net/browse/RMET-446

## Type of changes

- [ ] Fix (non-breaking change which fixes an issue)
- [x] Feature (non-breaking change which adds functionality)
- [ ] Refactor (cosmetic changes)
- [ ] Breaking change (change that would cause existing functionality to not work as expected)

## Platforms affected
- [ ] Android
- [ ] iOS
- [x] JavaScript

## Checklist

- [x] Pull request title follows the format `RNMT-XXXX <title>`
- [x] Code follows code style of this project
- [x] CHANGELOG.md file is correctly updated
- [x] Changes require an update to the documentation
	- [ ] Documentation has been updated accordingly

### CODE REVIEW

File: www/OSFileViewer.js [hunk 1]
```diff
@@ -1,5 +1,23 @@
 var exec = require('cordova/exec');
 
-exports.coolMethod = function (arg0, success, error) {
-    exec(success, error, 'OSFileViewer', 'coolMethod', [arg0]);
+exports.previewDocumentFromLocalPath = function (filePath, fileMIMEType, success, error) {
+    exec(success, error, 'OSFileViewer', 'previewDocumentFromLocalPath', [filePath, fileMIMEType]);
 };
+
+exports.previewDocumentFromUrl = function (url, fileMIMEType, success, error) {
```
Comment (vmfo22) [inline www/OSFileViewer.js:7]: Do we need the mimeType variable in the actions to open a URL?

**Review by vmfo22 — COMMENTED** [review]
Did you consider to have the validations, at this level. for the parameters received?

File: www/OSFileViewer.js (hunk 1 above)
Comment (alexgerardojacinto) [reply www/OSFileViewer.js:7]: We don't need it. We will remove this parameter and update the PR.

**Comment by alexgerardojacinto** [comment]
> Did you consider to have the validations, at this level. for the parameters received?

We have the validations on the native side. It will be easier to implement the unit tests there.

**Review by vmfo22 — APPROVED** [review]

**Review by CarlsCorrea — APPROVED** [review]

**Pull request merged**